stories/Steps.stories.js

```diff
@@ -33,3 +33,15 @@ export const Vertical = () => (
     collapse={boolean('collapse', false)}
   />
 );
+
+export const Clickable = () => (
+  <Steps
+    steps={steps}
+    step={number('step', 0)}
+    complete={boolean('complete', false)}
+    vertical={boolean('vertical', false)}
+    collapse={boolean('collapse', false)}
+    // eslint-disable-next-line no-alert
+    onStepClick={i => alert(`You clicked step ${i}!`)}
```

Comment:
Instead of doing an alert, we can use storybook actions and storybook will continuously log the function calls/params directly in the right side panel! (See [example](https://github.com/appfolio/react-gears/blob/master/stories/Table.stories.js#L110))

![image](https://user-images.githubusercontent.com/2031802/118325783-cfe7f700-b4b8-11eb-8d3d-15b71ae2cea3.png)

Reply:
Ah, neat! Figured there must be a better way. Switched it to use `action('index')`.

Reply:
🎉